src/vos/vos_space.c

```diff
@@ -19,11 +19,10 @@
 
 /* Extra space being reserved to deal with fragmentation issues */
 static inline daos_size_t
-get_frag_overhead(daos_size_t tot_size, int media, bool small_pool)
+get_frag_overhead(daos_size_t tot_size, int media)
```

Comment:
I suggest we keep this 'small_pool' for RDB for this moment.

Reply:
- [ ] Restore small_pool param for RDB

Reply:
Fixed with commit https://github.com/daos-stack/daos/pull/13013/commits/270805402e811b5f153193363cf4304b7dcc26f0#diff-d43f9e7f3650645f86cda7b2c96d5e007c6b0567bdce7807f25da2274fa5b42dL22